### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Ntwali-Josue**
### In this Pull Request, I: 

- [x] Add styles to the page

### CODE REVIEW

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Ntwali-Josue, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: The design is looking good.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [x] README file should have correct live link. currently it's linking to `page not found` because of the wrong link.
- [ ] I would recommend giving styling to a separate file instead of inline styling in the `src/components/BookItem.js` file. Inline styling is a bad practice as you can't use the things efficiently, instead you can give style to class and use it in several place.

- [ ] The design should be like the given one instead of writing the text only.

### Given design
Screenshot from 2021-10-02 03-51-40

### Your design
Screenshot from 2021-10-02 03-59-26
 





## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ntwali-Josue** [comment]
@Dipeshtwis Thanks for the reviews, I have worked on all of them as you suggested: 

- [x] Update Readme live demo link.
- [x] Remove all the inline styling and add styles in one file
- [x]  Update the design and add book progress circle.

File: src/App.css [hunk 1]
```diff
@@ -8,13 +8,20 @@
   margin: 0 10px;
 }
 
-/* Books */
-
 .book-btn {
-  margin: 5px 0;
+  /* margin: 5px 0; */
```
Comment (Dipeshtwis) [inline src/App.css:12]: [Optional] [ Highly Recommended]

- I would highly recommend removing the commented code to the codebase cleaner. Before doing merge please fix this.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @Ntwali-Josue, 

### Status: Approved ✔️

:+1:  You have implemented the request changes. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**



**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (Ntwali-Josue) [reply src/App.css:12]: Sure thanks for pointing this out. I am going to work on it 👍🏾

**Pull request merged**